### ISSUE TITLE
A few enhancement suggestions

### ISSUE THREAD

**Issue by somechazzy** [tedhinklater/Jellyfin-Featured-Content-Bar#35]
1. "Trailer" and "Play" buttons aren't relative to each other for centering. In my case I have many titles without a trailer, so the "Play" button remains aligned to the middle-right of the spotlight even if the "Trailer" button is missing.
2. A smooth transition between titles. Currently the spotlight just abruptly stops its animation and changes to previewing the new title without a fade-out for the previous one, nor a fade-in for the current. 
3. Overlay items (logo, genres, buttons etc..) take a little too long to display. While this is entirely subjective I still thought I should mention it.

Many thanks for your work.

**Comment by Harrenkyym** [comment]
More suggestions:

- To be able to use the ID of a collection to display its entire content.
- Following this thread, I've would suggest the transition to be kind of a "slideshow" where you can even click AND drag, that's way better for phones or tablets. (like here https://www.codingnepalweb.com/draggable-image-slider-html-css-javascript/)
- It is not accepting series right now. I've tried and it's like I haven't set any ID on the list. It would be cool if we can have series too.
- It would be great if we could add some kind of custom message (or even image). For example "It's Christmas time!" or "Oscar season!" in a corner or I don't know, but something that the users (for me it's my family) know why the movie is featured. When I say "or even image" I'm thinking about something like that, where if you have skills, you can custom the image like you want: 
![etiqueta-esquina-etiqueta-esquina-ilustracion-3d_118019-8762](https://github.com/user-attachments/assets/4a92c07f-598c-4a62-adff-a6c95c20cde6)
 - I think the rating tag is oversized... I don't know, but at least in Spain we are not that picky with that. (And in my particular case, in my family there are only adults, so I don't care a second about it)
 - That's how I see the fullscreen spotlight in my phone... And if I rotate the screen (which I never do it), the space between the image and my collections still there.
![thumbnail_1000100545](https://github.com/user-attachments/assets/6d534cea-4bd8-4005-b411-cd14650fd9c4)


I think that's all.

**Comment by tedhinklater** [comment]
the trailer/play buttons have been justified and the opacity timing adjusted

**Comment by tedhinklater** [comment]
> * It is not accepting series right now. I've tried and it's like I haven't set any ID on the list. It would be cool if we can have series too.

It should definitely accept series/shows. Fullscreen has been redone hopefully it's a bit better

**Comment by Harrenkyym** [comment]
> > * It is not accepting series right now. I've tried and it's like I haven't set any ID on the list. It would be cool if we can have series too.
> 
> It should definitely accept series/shows. Fullscreen has been redone hopefully it's a bit better

Yep, I noticed it does! The thing is I discovered it doesn't accept content without logo (movie or series) and the series I wanted to show didn't have a logo. I set a good one and the series was displayed immediately.
The bad thing is that it is in english. It says "1 Season" instead of "1 temporada", it should be in spanish (in my case, obviously). Is there something that must be translated somewhere?

And I've noticed too something weird. You can't "go back" from the first content displayed. You can go forward and see again the first content, but if you go back you'll always get to the first element displayed (for the first time, when everything loads) then you can't go back anymore... I think it should be an endless carousel. :D

**Comment by tedhinklater** [comment]
the bar definitely doesn't use ffmpeg for anything, are you sure your users aren't transcoding content? 
You can click the (i) under Active Devices on the Dashboard to check if the users are Direct Playing or Transcoding
![Screenshot 2025-01-10 124537](https://github.com/user-attachments/assets/7e764714-fead-4f21-a1bd-840f5f00cc18)

**Comment by tedhinklater** [comment]
collections/albums work now, swiping too on mobile

**Comment by tedhinklater** [comment]
I'm trying to get language localization working, in the meantime you can edit spotlight.html and replace ```Season$``` with ```Temporada$```

**Comment by tedhinklater** [comment]
The genres section just displays the content's genres metadata, so if it's written in English in the metadata the bar will display that.

I can't recreate the ```null item``` bug you described, I'll keep looking.

I see what you're saying about the scrolling, I'll fix that now. 

Make sure your collection has a Backdrop image and Logo image.

I've changed the Age Rating to be hidden after a second or so

![genres](https://github.com/user-attachments/assets/4bea32bd-dda8-4fd0-a675-330f55cea088)

**Comment by Harrenkyym** [comment]
- Yep, I just discovered that genres of series en tmdb are shit, less than in movies and not translated...

- The null thing happens when the content on the list is not displaying (maybe because there is no logo), I discovered that it just happens with not displaying content, it makes Spotlight disappear at some point.

I don't see that age thing happening on fullscreen at least. Anyway, I think there should be an option to make it important or not, because I'm pretty sure there are a lot of people who needs it.

- I promise I've used a collection with backdrop image and logo, where movies inside have backdrop image and logo... I can't see them.

**Comment by tedhinklater** [comment]
Just so we're on the same page: you downloaded the latest spotlight.html (fullscreen) and pasted the latest home-html.chunk.js (fullscreen) and have the fullscreen css? and your external monitor is 1080p 16:9 ?

**Comment by Harrenkyym** [comment]
Yes, but as I said, it only happens because I set my jellyfin site to display at the 80% of the normal size. But a few versions ago it was perfect with that 80%.
